samples/Provider/SampleEntityFrameworkProvider/ProviderManifest.cs

```diff
@@ -0,0 +1,617 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.
```

Comment:
This seems to be a merge mistake. This is not my file. Ok, it was modified by me, but it seems that later it was removed. So you should remove this file from the commit (and the following 2 files, too)